### ISSUE TITLE
Is Localtuya dead?

### ISSUE THREAD

**Issue by vivdesign** [rospogrigio/localtuya#1699]
There has be no updates for over an year. Is this project dead?

**Comment by neildsb** [comment]
maybe try this fork? https://github.com/xZetsubou/hass-localtuya

**Comment by darth-aragoth** [comment]
Yes, but you won't be able to go back. I'm restoring from backup right now...
